website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -269,39 +286,51 @@ helm install \
         --set autoupdate.enabled=true
     ```
 
-### Manual Update
+### Manual update
 
-* To manually update the Appsmith container image to the latest release, run the command: `kubectl rollout restart statefulset appsmith`
+To manually update the Appsmith container image to the latest release, run the command: 
 
-## Nginx ingress controller compression (Gzip)
+```bash
+kubectl rollout restart statefulset appsmith
+```
 
-In the default installation, the Nginx ingress controller will not have the data compression configured. Please follow the following steps to enable it.
+## Uninstall
+You can uninstall the <mark>`appsmith`</mark> release. The command uninstalls the release and removes Kubernetes resources associated with the chart.
 
-* Edit the `configMap` for `ingress-nginx-controller` by using the below command
+```bash
+helm list
+NAME                       NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
+#highlight-next-line
+appsmith-1631069261        default         1               2021-09-09 11:24:40.152766 +0700 +07    deployed        appsmith-1.3.0  1.16.0
 
+helm uninstall appsmith-1631069261
 ```
+
+## Enable Ingress data compression
```

Comment:
This should be 'Compress Nginx Ingress controller'

Reply:
As discussed - I will add - `Enable data compression for Ingress controller`